
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.5.3, mkdocs-material-9.5.15" name="generator"/>
<title>Class: CellDissociation - model</title>
<link href="../assets/stylesheets/main.7e359304.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
<script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
</head>
<body dir="ltr">
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#class-celldissociation">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header md-header--shadow" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="model" class="md-header__button md-logo" data-md-component="logo" href=".." title="model">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            model
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              Class: CellDissociation
            
          </span>
</div>
</div>
</div>
<script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"></path></svg>
</label>
<nav aria-label="Search" class="md-search__options">
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" title="Clear" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"></path></svg>
</button>
</nav>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list" role="presentation"></ol>
</div>
</div>
</div>
</div>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="model" class="md-nav__button md-logo" data-md-component="logo" href=".." title="model">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
    model
  </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
<span class="md-ellipsis">
    Home
  </span>
</a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#inheritance">
<span class="md-ellipsis">
      Inheritance
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#slots">
<span class="md-ellipsis">
      Slots
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#usages">
<span class="md-ellipsis">
      Usages
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#identifier-and-mapping-information">
<span class="md-ellipsis">
      Identifier and Mapping Information
    </span>
</a>
<nav aria-label="Identifier and Mapping Information" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#schema-source">
<span class="md-ellipsis">
      Schema Source
    </span>
</a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#mappings">
<span class="md-ellipsis">
      Mappings
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#linkml-source">
<span class="md-ellipsis">
      LinkML Source
    </span>
</a>
<nav aria-label="LinkML Source" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#direct">
<span class="md-ellipsis">
      Direct
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#induced">
<span class="md-ellipsis">
      Induced
    </span>
</a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<h1 id="class-celldissociation">Class: CellDissociation</h1>
<p><em>The process of generating dissociated cells from an input tissue sample. This process could also introduce a tissue-source barcode (eg cell hashing), allowing mixing of cell dissociation samples at the cell barcoding step.</em></p>
<p>URI: <a href="https://identifiers.org/brain-bican/vocab/CellDissociation">bican:CellDissociation</a></p>
<div class="mermaid"> classDiagram
    class CellDissociation
      ProvActivity &lt;|-- CellDissociation
      Procedure &lt;|-- CellDissociation

      CellDissociation : category

      CellDissociation : deprecated

      CellDissociation : description

      CellDissociation : full_name

      CellDissociation : has_attribute

          CellDissociation --&gt; attribute : has_attribute

      CellDissociation : id

      CellDissociation : iri

      CellDissociation : name

      CellDissociation : named_thing_category

      CellDissociation : process_date

      CellDissociation : provided_by

      CellDissociation : synonym

      CellDissociation : type

      CellDissociation : used

          CellDissociation --&gt; TissueSample : used

      CellDissociation : was_guided_by

          CellDissociation --&gt; ProvEntity : was_guided_by

      CellDissociation : xref


</div>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../Entity/">Entity</a><ul>
<li><a href="../NamedThing/">NamedThing</a><ul>
<li><a href="../Procedure/">Procedure</a> [ <a href="../ActivityAndBehavior/">ActivityAndBehavior</a>]<ul>
<li><strong>CellDissociation</strong> [ <a href="../ProvActivity/">ProvActivity</a>]</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Name</th>
<th>Cardinality and Range</th>
<th>Description</th>
<th>Inheritance</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../used/">used</a></td>
<td>0..* <br/> <a href="../TissueSample/">TissueSample</a></td>
<td>The input tissue sample(s) from which the dissociated cell sample was derived...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../process_date/">process_date</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Date of cell dissociation process</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../was_guided_by/">was_guided_by</a></td>
<td>0..1 <br/> <a href="../ProvEntity/">ProvEntity</a></td>
<td>Guidance is the influence of an entity on an activity</td>
<td><a href="../ProvActivity/">ProvActivity</a></td>
</tr>
<tr>
<td><a href="../id/">id</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>A unique identifier for an entity</td>
<td><a href="../Entity/">Entity</a>, <a href="../Procedure/">Procedure</a>, <a href="../NamedThing/">NamedThing</a></td>
</tr>
<tr>
<td><a href="../iri/">iri</a></td>
<td>0..1 <br/> <a href="../IriType/">IriType</a></td>
<td>An IRI for an entity</td>
<td><a href="../Entity/">Entity</a>, <a href="../Procedure/">Procedure</a>, <a href="../NamedThing/">NamedThing</a></td>
</tr>
<tr>
<td><a href="../type/">type</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td></td>
<td><a href="../Entity/">Entity</a>, <a href="../Procedure/">Procedure</a>, <a href="../NamedThing/">NamedThing</a></td>
</tr>
<tr>
<td><a href="../name/">name</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td></td>
<td><a href="../Entity/">Entity</a>, <a href="../Procedure/">Procedure</a>, <a href="../NamedThing/">NamedThing</a></td>
</tr>
<tr>
<td><a href="../description/">description</a></td>
<td>0..1 <br/> <a href="../NarrativeText/">NarrativeText</a></td>
<td>a human-readable description of an entity</td>
<td><a href="../Entity/">Entity</a>, <a href="../Procedure/">Procedure</a>, <a href="../NamedThing/">NamedThing</a></td>
</tr>
<tr>
<td><a href="../has_attribute/">has_attribute</a></td>
<td>0..* <br/> <a href="../Attribute/">Attribute</a></td>
<td>connects any entity to an attribute</td>
<td><a href="../Entity/">Entity</a>, <a href="../Procedure/">Procedure</a>, <a href="../NamedThing/">NamedThing</a></td>
</tr>
<tr>
<td><a href="../deprecated/">deprecated</a></td>
<td>0..1 <br/> <a href="../Boolean/">Boolean</a></td>
<td>A boolean flag indicating that an entity is no longer considered current or v...</td>
<td><a href="../Entity/">Entity</a>, <a href="../Procedure/">Procedure</a>, <a href="../NamedThing/">NamedThing</a></td>
</tr>
<tr>
<td><a href="../provided_by/">provided_by</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>The value in this node property represents the knowledge provider that create...</td>
<td><a href="../Procedure/">Procedure</a>, <a href="../NamedThing/">NamedThing</a></td>
</tr>
<tr>
<td><a href="../xref/">xref</a></td>
<td>0..* <br/> <a href="../Uriorcurie/">Uriorcurie</a></td>
<td>A database cross reference or alternative identifier for a NamedThing or edge...</td>
<td><a href="../Procedure/">Procedure</a>, <a href="../NamedThing/">NamedThing</a></td>
</tr>
<tr>
<td><a href="../full_name/">full_name</a></td>
<td>0..1 <br/> <a href="../LabelType/">LabelType</a></td>
<td>a long-form human readable name for a thing</td>
<td><a href="../Procedure/">Procedure</a>, <a href="../NamedThing/">NamedThing</a></td>
</tr>
<tr>
<td><a href="../synonym/">synonym</a></td>
<td>0..* <br/> <a href="../LabelType/">LabelType</a></td>
<td>Alternate human-readable names for a thing</td>
<td><a href="../Procedure/">Procedure</a>, <a href="../NamedThing/">NamedThing</a></td>
</tr>
<tr>
<td><a href="../named_thing_category/">named_thing_category</a></td>
<td>1..* <br/> <a href="../CategoryType/">CategoryType</a></td>
<td>Name of the high level ontology class in which this entity is categorized</td>
<td><a href="../Procedure/">Procedure</a>, <a href="../NamedThing/">NamedThing</a></td>
</tr>
<tr>
<td><a href="../category/">category</a></td>
<td>1..* <br/> <a href="../CategoryType/">CategoryType</a></td>
<td>Name of the high level ontology class in which this entity is categorized</td>
<td><a href="../Entity/">Entity</a></td>
</tr>
</tbody>
</table>
<h2 id="usages">Usages</h2>
<table>
<thead>
<tr>
<th>used by</th>
<th>used in</th>
<th>type</th>
<th>used</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../DissociatedCellSample/">DissociatedCellSample</a></td>
<td><a href="../was_generated_by/">was_generated_by</a></td>
<td>range</td>
<td><a href="./">CellDissociation</a></td>
</tr>
</tbody>
</table>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>
<h3 id="schema-source">Schema Source</h3>
<ul>
<li>from schema: https://identifiers.org/brain-bican/kb-model</li>
</ul>
<h2 id="mappings">Mappings</h2>
<table>
<thead>
<tr>
<th>Mapping Type</th>
<th>Mapped Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>self</td>
<td>bican:CellDissociation</td>
</tr>
<tr>
<td>native</td>
<td>bican:CellDissociation</td>
</tr>
</tbody>
</table>
<h2 id="linkml-source">LinkML Source</h2>
<!-- TODO: investigate https://stackoverflow.com/questions/37606292/how-to-create-tabbed-code-blocks-in-mkdocs-or-sphinx -->
<h3 id="direct">Direct</h3>
<details>
<pre><code class="language-yaml">name: CellDissociation
description: The process of generating dissociated cells from an input tissue sample.
  This process could also introduce a tissue-source barcode (eg cell hashing), allowing
  mixing of cell dissociation samples at the cell barcoding step.
in_subset:
- bican, library_generation
from_schema: https://identifiers.org/brain-bican/kb-model
is_a: procedure
mixins:
- ProvActivity
slots:
- used
slot_usage:
  used:
    name: used
    description: The input tissue sample(s) from which the dissociated cell sample
      was derived from.
    from_schema: https://identifiers.org/brain-bican/kb-model
    multivalued: true
    domain_of:
    - DissectionRoiDelineation
    - TissueDissection
    - CellDissociation
    - CellEnrichment
    - EnrichedCellSampleSplitting
    - CellBarcoding
    - CdnaAmplification
    - LibraryConstruction
    - LibraryPooling
    - ProvActivity
    range: TissueSample
attributes:
  process_date:
    name: process_date
    description: Date of cell dissociation process.
    from_schema: https://identifiers.org/brain-bican/kb-model
    exact_mappings:
    - NIMP:PD-BUBUFE27
    domain_of:
    - CellDissociation
    - CellEnrichment
    - CellBarcoding
    - CdnaAmplification
    - LibraryConstruction
    - LibraryPooling

</code></pre>
</details>
<h3 id="induced">Induced</h3>
<details>
<pre><code class="language-yaml">name: CellDissociation
description: The process of generating dissociated cells from an input tissue sample.
  This process could also introduce a tissue-source barcode (eg cell hashing), allowing
  mixing of cell dissociation samples at the cell barcoding step.
in_subset:
- bican, library_generation
from_schema: https://identifiers.org/brain-bican/kb-model
is_a: procedure
mixins:
- ProvActivity
slot_usage:
  used:
    name: used
    description: The input tissue sample(s) from which the dissociated cell sample
      was derived from.
    from_schema: https://identifiers.org/brain-bican/kb-model
    multivalued: true
    domain_of:
    - DissectionRoiDelineation
    - TissueDissection
    - CellDissociation
    - CellEnrichment
    - EnrichedCellSampleSplitting
    - CellBarcoding
    - CdnaAmplification
    - LibraryConstruction
    - LibraryPooling
    - ProvActivity
    range: TissueSample
attributes:
  process_date:
    name: process_date
    description: Date of cell dissociation process.
    from_schema: https://identifiers.org/brain-bican/kb-model
    exact_mappings:
    - NIMP:PD-BUBUFE27
    alias: process_date
    owner: CellDissociation
    domain_of:
    - CellDissociation
    - CellEnrichment
    - CellBarcoding
    - CdnaAmplification
    - LibraryConstruction
    - LibraryPooling
    range: string
  used:
    name: used
    description: The input tissue sample(s) from which the dissociated cell sample
      was derived from.
    from_schema: https://identifiers.org/brain-bican/kb-model
    rank: 1000
    slot_uri: prov:used
    multivalued: true
    alias: used
    owner: CellDissociation
    domain_of:
    - DissectionRoiDelineation
    - TissueDissection
    - CellDissociation
    - CellEnrichment
    - EnrichedCellSampleSplitting
    - CellBarcoding
    - CdnaAmplification
    - LibraryConstruction
    - LibraryPooling
    - ProvActivity
    range: TissueSample
  was_guided_by:
    name: was_guided_by
    description: Guidance is the influence of an entity on an activity. This entity
      is known as an influencer, and the activity is influenced by the influencer.
    from_schema: https://identifiers.org/brain-bican/kb-model
    rank: 1000
    alias: was_guided_by
    owner: CellDissociation
    domain_of:
    - TissueDissection
    - ProvActivity
    range: ProvEntity
  id:
    name: id
    definition_uri: https://w3id.org/biolink/id
    description: A unique identifier for an entity. Must be either a CURIE shorthand
      for a URI or a complete URI
    in_subset:
    - translator_minimal
    from_schema: https://identifiers.org/brain-bican/kb-model
    exact_mappings:
    - AGRKB:primaryId
    - gff3:ID
    - gpi:DB_Object_ID
    rank: 1000
    domain: entity
    slot_uri: biolink:id
    identifier: true
    alias: id
    owner: CellDissociation
    domain_of:
    - ontology class
    - entity
    - attribute
    - named thing
    - taxonomic rank
    - organism taxon
    - physical entity
    - activity
    - procedure
    - material sample
    - biological entity
    - gene
    - genome
    range: string
    required: true
  iri:
    name: iri
    definition_uri: https://w3id.org/biolink/iri
    description: An IRI for an entity. This is determined by the id using expansion
      rules.
    in_subset:
    - translator_minimal
    - samples
    from_schema: https://identifiers.org/brain-bican/kb-model
    exact_mappings:
    - WIKIDATA_PROPERTY:P854
    rank: 1000
    slot_uri: biolink:iri
    alias: iri
    owner: CellDissociation
    domain_of:
    - attribute
    - entity
    - named thing
    - organism taxon
    - physical entity
    - activity
    - procedure
    - material sample
    - biological entity
    - gene
    - genome
    range: iri type
  type:
    name: type
    definition_uri: https://w3id.org/biolink/type
    from_schema: https://identifiers.org/brain-bican/kb-model
    mappings:
    - rdf:type
    exact_mappings:
    - gff3:type
    - gpi:DB_Object_Type
    rank: 1000
    domain: entity
    slot_uri: rdf:type
    multivalued: true
    alias: type
    owner: CellDissociation
    domain_of:
    - entity
    - attribute
    - named thing
    - organism taxon
    - physical entity
    - activity
    - procedure
    - material sample
    - biological entity
    - gene
    - genome
    range: string
  name:
    name: name
    from_schema: https://identifiers.org/brain-bican/kb-model
    rank: 1000
    alias: name
    owner: CellDissociation
    domain_of:
    - Donor
    - BrainSlab
    - TissueSample
    - DissociatedCellSample
    - EnrichedCellSample
    - BarcodedCellSample
    - AmplifiedCdna
    - Library
    - LibraryAliquot
    - LibraryPool
    - DissectionRoiPolygon
    - entity
    - named thing
    - organism taxon
    - physical entity
    - activity
    - procedure
    - material sample
    - biological entity
    - gene
    - genome
    range: string
  description:
    name: description
    definition_uri: https://w3id.org/biolink/description
    description: a human-readable description of an entity
    in_subset:
    - translator_minimal
    from_schema: https://identifiers.org/brain-bican/kb-model
    aliases:
    - definition
    mappings:
    - dct:description
    exact_mappings:
    - IAO:0000115
    - skos:definitions
    narrow_mappings:
    - gff3:Description
    rank: 1000
    slot_uri: dct:description
    alias: description
    owner: CellDissociation
    domain_of:
    - entity
    - attribute
    - named thing
    - organism taxon
    - physical entity
    - activity
    - procedure
    - material sample
    - biological entity
    - gene
    - genome
    range: narrative text
  has attribute:
    name: has attribute
    definition_uri: https://w3id.org/biolink/has_attribute
    description: connects any entity to an attribute
    in_subset:
    - samples
    from_schema: https://identifiers.org/brain-bican/kb-model
    exact_mappings:
    - SIO:000008
    close_mappings:
    - OBI:0001927
    narrow_mappings:
    - OBAN:association_has_subject_property
    - OBAN:association_has_object_property
    - CPT:has_possibly_included_panel_element
    - DRUGBANK:category
    - EFO:is_executed_in
    - HANCESTRO:0301
    - LOINC:has_action_guidance
    - LOINC:has_adjustment
    - LOINC:has_aggregation_view
    - LOINC:has_approach_guidance
    - LOINC:has_divisor
    - LOINC:has_exam
    - LOINC:has_method
    - LOINC:has_modality_subtype
    - LOINC:has_object_guidance
    - LOINC:has_scale
    - LOINC:has_suffix
    - LOINC:has_time_aspect
    - LOINC:has_time_modifier
    - LOINC:has_timing_of
    - NCIT:R88
    - NCIT:eo_disease_has_property_or_attribute
    - NCIT:has_data_element
    - NCIT:has_pharmaceutical_administration_method
    - NCIT:has_pharmaceutical_basic_dose_form
    - NCIT:has_pharmaceutical_intended_site
    - NCIT:has_pharmaceutical_release_characteristics
    - NCIT:has_pharmaceutical_state_of_matter
    - NCIT:has_pharmaceutical_transformation
    - NCIT:is_qualified_by
    - NCIT:qualifier_applies_to
    - NCIT:role_has_domain
    - NCIT:role_has_range
    - INO:0000154
    - HANCESTRO:0308
    - OMIM:has_inheritance_type
    - orphanet:C016
    - orphanet:C017
    - RO:0000053
    - RO:0000086
    - RO:0000087
    - SNOMED:has_access
    - SNOMED:has_clinical_course
    - SNOMED:has_count_of_base_of_active_ingredient
    - SNOMED:has_dose_form_administration_method
    - SNOMED:has_dose_form_release_characteristic
    - SNOMED:has_dose_form_transformation
    - SNOMED:has_finding_context
    - SNOMED:has_finding_informer
    - SNOMED:has_inherent_attribute
    - SNOMED:has_intent
    - SNOMED:has_interpretation
    - SNOMED:has_laterality
    - SNOMED:has_measurement_method
    - SNOMED:has_method
    - SNOMED:has_priority
    - SNOMED:has_procedure_context
    - SNOMED:has_process_duration
    - SNOMED:has_property
    - SNOMED:has_revision_status
    - SNOMED:has_scale_type
    - SNOMED:has_severity
    - SNOMED:has_specimen
    - SNOMED:has_state_of_matter
    - SNOMED:has_subject_relationship_context
    - SNOMED:has_surgical_approach
    - SNOMED:has_technique
    - SNOMED:has_temporal_context
    - SNOMED:has_time_aspect
    - SNOMED:has_units
    - UMLS:has_structural_class
    - UMLS:has_supported_concept_property
    - UMLS:has_supported_concept_relationship
    - UMLS:may_be_qualified_by
    rank: 1000
    domain: entity
    slot_uri: biolink:has_attribute
    multivalued: true
    alias: has_attribute
    owner: CellDissociation
    domain_of:
    - entity
    - attribute
    - named thing
    - organism taxon
    - physical entity
    - activity
    - procedure
    - material sample
    - biological entity
    - gene
    - genome
    range: attribute
  deprecated:
    name: deprecated
    definition_uri: https://w3id.org/biolink/deprecated
    description: A boolean flag indicating that an entity is no longer considered
      current or valid.
    from_schema: https://identifiers.org/brain-bican/kb-model
    exact_mappings:
    - oboInOwl:ObsoleteClass
    rank: 1000
    slot_uri: biolink:deprecated
    alias: deprecated
    owner: CellDissociation
    domain_of:
    - entity
    - attribute
    - named thing
    - organism taxon
    - physical entity
    - activity
    - procedure
    - material sample
    - biological entity
    - gene
    - genome
    range: boolean
  provided by:
    name: provided by
    definition_uri: https://w3id.org/biolink/provided_by
    description: The value in this node property represents the knowledge provider
      that created or assembled the node and all of its attributes.  Used internally
      to represent how a particular node made its way into a knowledge provider or
      graph.
    from_schema: https://identifiers.org/brain-bican/kb-model
    rank: 1000
    is_a: node property
    domain: named thing
    slot_uri: biolink:provided_by
    multivalued: true
    alias: provided_by
    owner: CellDissociation
    domain_of:
    - named thing
    - attribute
    - organism taxon
    - physical entity
    - activity
    - procedure
    - material sample
    - biological entity
    - gene
    - genome
    range: string
  xref:
    name: xref
    definition_uri: https://w3id.org/biolink/xref
    description: A database cross reference or alternative identifier for a NamedThing
      or edge between two NamedThings.  This property should point to a database record
      or webpage that supports the existence of the edge, or gives more detail about
      the edge. This property can be used on a node or edge to provide multiple URIs
      or CURIE cross references.
    in_subset:
    - translator_minimal
    from_schema: https://identifiers.org/brain-bican/kb-model
    aliases:
    - dbxref
    - Dbxref
    - DbXref
    - record_url
    - source_record_urls
    narrow_mappings:
    - gff3:Dbxref
    - gpi:DB_Xrefs
    rank: 1000
    domain: named thing
    slot_uri: biolink:xref
    multivalued: true
    alias: xref
    owner: CellDissociation
    domain_of:
    - named thing
    - gene
    - attribute
    - organism taxon
    - physical entity
    - activity
    - procedure
    - material sample
    - biological entity
    - genome
    range: uriorcurie
  full name:
    name: full name
    definition_uri: https://w3id.org/biolink/full_name
    description: a long-form human readable name for a thing
    from_schema: https://identifiers.org/brain-bican/kb-model
    rank: 1000
    is_a: node property
    domain: named thing
    slot_uri: biolink:full_name
    alias: full_name
    owner: CellDissociation
    domain_of:
    - named thing
    - attribute
    - organism taxon
    - physical entity
    - activity
    - procedure
    - material sample
    - biological entity
    - gene
    - genome
    range: label type
  synonym:
    name: synonym
    definition_uri: https://w3id.org/biolink/synonym
    description: Alternate human-readable names for a thing
    in_subset:
    - translator_minimal
    from_schema: https://identifiers.org/brain-bican/kb-model
    aliases:
    - alias
    narrow_mappings:
    - skos:altLabel
    - gff3:Alias
    - AGRKB:synonyms
    - gpi:DB_Object_Synonyms
    - HANCESTRO:0330
    - IAO:0000136
    - RXNORM:has_tradename
    rank: 1000
    is_a: node property
    domain: named thing
    slot_uri: biolink:synonym
    multivalued: true
    alias: synonym
    owner: CellDissociation
    domain_of:
    - named thing
    - attribute
    - organism taxon
    - physical entity
    - activity
    - procedure
    - material sample
    - biological entity
    - gene
    - genome
    range: label type
  named thing_category:
    name: named thing_category
    definition_uri: https://w3id.org/biolink/category
    description: Name of the high level ontology class in which this entity is categorized.
      Corresponds to the label for the biolink entity type class. In a neo4j database
      this MAY correspond to the neo4j label tag. In an RDF database it should be
      a biolink model class URI. This field is multi-valued. It should include values
      for ancestors of the biolink class; for example, a protein such as Shh would
      have category values `biolink:Protein`, `biolink:GeneProduct`, `biolink:MolecularEntity`.
      In an RDF database, nodes will typically have an rdf:type triples. This can
      be to the most specific biolink class, or potentially to a class more specific
      than something in biolink. For example, a sequence feature `f` may have a rdf:type
      assertion to a SO class such as TF_binding_site, which is more specific than
      anything in biolink. Here we would have categories {biolink:GenomicEntity, biolink:MolecularEntity,
      biolink:NamedThing}
    in_subset:
    - translator_minimal
    from_schema: https://identifiers.org/brain-bican/kb-model
    rank: 1000
    is_a: category
    domain: named thing
    slot_uri: biolink:category
    multivalued: true
    designates_type: true
    alias: category
    owner: CellDissociation
    domain_of:
    - named thing
    - attribute
    - organism taxon
    - physical entity
    - activity
    - procedure
    - material sample
    - biological entity
    - gene
    - genome
    is_class_field: true
    is_usage_slot: true
    usage_slot_name: category
    range: category type
    required: true
    pattern: ^biolink:[A-Z][A-Za-z]+$
  category:
    name: category
    definition_uri: https://w3id.org/biolink/category
    description: Name of the high level ontology class in which this entity is categorized.
      Corresponds to the label for the biolink entity type class. In a neo4j database
      this MAY correspond to the neo4j label tag. In an RDF database it should be
      a biolink model class URI. This field is multi-valued. It should include values
      for ancestors of the biolink class; for example, a protein such as Shh would
      have category values `biolink:Protein`, `biolink:GeneProduct`, `biolink:MolecularEntity`.
      In an RDF database, nodes will typically have an rdf:type triples. This can
      be to the most specific biolink class, or potentially to a class more specific
      than something in biolink. For example, a sequence feature `f` may have a rdf:type
      assertion to a SO class such as TF_binding_site, which is more specific than
      anything in biolink. Here we would have categories {biolink:GenomicEntity, biolink:MolecularEntity,
      biolink:NamedThing}
    from_schema: https://identifiers.org/brain-bican/kb-model
    rank: 1000
    is_a: type
    domain: entity
    slot_uri: biolink:category
    multivalued: true
    designates_type: true
    alias: category
    owner: CellDissociation
    domain_of:
    - entity
    is_class_field: true
    range: category type
    required: true
    pattern: ^biolink:[A-Z][A-Za-z]+$

</code></pre>
</details>
</article>
</div>
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
</div>
</main>
<footer class="md-footer">
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
      Material for MkDocs
    </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": [], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
<script src="../assets/javascripts/bundle.bd41221c.min.js"></script>
<script type="module">import mermaid from "https://unpkg.com/mermaid@10.4.0/dist/mermaid.esm.min.mjs";
mermaid.initialize({});</script></body>
</html>